spec/rubocop/cop/style/dot_position_spec.rb

```diff
@@ -71,6 +71,15 @@
                                 'something',
                                 '  .method_name'].join("\n"))
     end
+
+    context 'when there is an intervening line comment' do
```

Comment:
I'd add a test about a blank line as well, as the effect of doing auto-correct then would be the same.